stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -565,6 +613,37 @@ public struct Unsafe${Mutable}RawBufferPointer
   }
   %  end # mutable
 
+  /// Binds this buffer’s memory to the specified type and returns a typed buffer 
+  /// of the bound memory.
+  ///
+  /// Use the `bindMemory(to:)` method to bind the memory referenced
+  /// by this buffer to the type `T`. The memory must be uninitialized or
+  /// initialized to a type that is layout compatible with `T`. If the memory
+  /// is uninitialized, it is still uninitialized after being bound to `T`.
+  ///
+  /// - Warning: A memory location may only be bound to one type at a time. The
+  ///   behavior of accessing memory as a type unrelated to its bound type is
+  ///   undefined.
+  ///
+  /// - Parameters:
+  ///   - type: The type `T` to bind the memory to.
+  /// - Returns: A typed buffer of the newly bound memory. The memory in this
+  ///   region is bound to `T`, but has not been modified in any other way.
+  ///   The typed buffer references `self.count / MemoryLayout<T>.stride` instances of `T`.
+  @_transparent
```

Comment:
Should this be `@_inlineable` as well?

Reply:
isn’t @_transparent stronger than @_inlineable?

Reply:
I noticed there were a few places where Max used both annotations in that other PR, which is why I’m wondering if it’s necessary here. I was of the same understanding as you, but now I’m not so sure.

Reply:
i’m copying the 

    @_inlineable // FIXME(sil-serialize-all)
    @_transparent